CardParts.podspec

```diff
@@ -28,9 +28,9 @@ CardParts is an iOS Card UI framework that uses MVVM and automatic data binding
   s.swift_version = '4.2'
```

Comment:
Do we need to bump to `5.0`?
```suggestion
  s.swift_version = '5.0'
```

Reply:
Duh, totally. There might be some more changes now I expect.